### PR TITLE
Adding script and java file to automatically setup the pom files for those using iFixFlakies.

### PR DESCRIPTION
**Pull request by winglam**

### CODE REVIEW

File: pom-modify/modify-project.sh [hunk 1]
```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+if [[ $1 == "" ]]; then
+    echo "arg1 - the path to the project, where high-level pom.xml is"
```
Comment (august782) [inline pom-modify/modify-project.sh:6]: Maybe say "the path to the project, the directory where the high-level pom.xml is"

File: pom-modify/modify-project.sh [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    echo "arg2 - (Optional) Custom version for the artifact (e.g., 1.0.1, 1.0.2-SNAPSHOT)"
```
Comment (august782) [inline pom-modify/modify-project.sh]: Should it say the default version?

File: pom-modify/modify-project.sh (hunk 1 above)
Comment (winglam) [reply pom-modify/modify-project.sh:6]: Done

Comment (august782) [reply pom-modify/modify-project.sh:6]: It doesn't seem the change propagated. After this small change, should then be fine to merge.

**Review by august782 — APPROVED** [review]
There's a note about how the message for the argument does not say it is the path to the directory, but it is very minor, so should still be fine to merge.

**Pull request merged**